v4_notes.md

```diff
@@ -0,0 +1,4 @@
+# Notes for Version 4
+
+`reset_contexts` and `slice_coding_mode` were defined in a former version of the spec, it could be peek for a former version (<2018-01-04), commit d712fd1.
```

Comment:
You could give the link to the commit.

Reply:
chicken and egg: the reference Git repo is https://git.videolan.org/?p=ffmpeg.git and it does not have the commit yet.
Same for https://github.com/FFmpeg/FFV1 (a mirror) master branch, the only link available is in the PR https://github.com/FFmpeg/FFV1/pull/93/commits/0fe8b02af39a681e8862b279b2269bae0fc397b6 and I am reluctant to link to a GitHub PR commit.

Reply:
Ah, thanks, understood! I was thinking that a live link is more useful…